### PR TITLE
ADded AnchorKit & 2 more KitProperties

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/kotlin/net/axay/kotlinkitapi/api/KitProperties.kt [hunk 1]
```diff
@@ -7,6 +7,10 @@ import io.github.config4k.extract
 import net.axay.kotlinkitapi.config.ConfigManager
 import net.axay.kspigot.extensions.bukkit.warn
 import net.axay.kspigot.extensions.server
+import org.bukkit.Particle
+import org.bukkit.Sound
+import org.bukkit.potion.PotionEffect
+import org.bukkit.potion.PotionEffectType
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/kotlinkitapi/api/KitProperties.kt]: remove unused imports

File: src/main/kotlin/net/axay/kotlinkitapi/api/KitProperties.kt [hunk 2]
```diff
@@ -53,4 +57,6 @@ abstract class KitProperties {
     fun float(default: Float) = any(default)
     fun double(default: Double) = any(default)
     fun string(default: String) = any(default)
+    fun sound(default: Sound) = any(default)
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/kotlinkitapi/api/KitProperties.kt:59]: add a new line before these functions

File: src/main/kotlin/net/axay/kotlinkitapi/implementation/Anchor.kt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+val Anchor = Kit("Anchor", ::AnchorProperties) {
+    kitPlayerEvent<EntityDamageByEntityEvent>({ it.damager as? Player }) { it, damager ->
+        if (it.entity !is LivingEntity) return@kitPlayerEvent
+        val target = it.entity as LivingEntity
+        task(delay = 1L, howOften = 1L) {
+            target.velocity = Vector(0, 0, 0)
+        }
+        damager.playSound(target.location, this.kit.properties.sound, this.kit.properties.loudness, 1f)
+    }
+}
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/kotlinkitapi/implementation/Anchor.kt]: add a new line at the end of the file

File: src/main/kotlin/net/axay/kotlinkitapi/implementation/Anchor.kt [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import net.axay.kspigot.runnables.task
+import org.bukkit.Sound
+import org.bukkit.entity.LivingEntity
+import org.bukkit.entity.Player
+import org.bukkit.event.entity.EntityDamageByEntityEvent
+import org.bukkit.util.Vector
+
+class AnchorProperties : KitProperties() {
+    val sound by sound(Sound.BLOCK_ANVIL_PLACE)
+    val loudness by float(0.2f)
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/kotlinkitapi/implementation/Anchor.kt]: do you mean "volume"?

**Review by jakobkmar — CHANGES_REQUESTED** [review]
:+1:

File: src/main/kotlin/net/axay/kotlinkitapi/implementation/Anchor.kt (hunk 4 above)
Comment (ghost) [reply src/main/kotlin/net/axay/kotlinkitapi/implementation/Anchor.kt]: bruh

**Review by jakobkmar — APPROVED** [review]

**Pull request merged**